samples/tab-sso/nodejs/src/server/tabs.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+const fetch = require("node-fetch");
+const querystring = require("querystring");
+var config = require('config');
+const msal = require('@azure/msal-node');
+
+module.exports.setup = function (app) {
+  var express = require('express')
+
+  // Configure the view engine, views folder and the statics path
+  // Use the JSON middleware
+  app.use(express.json());
+
+  // Setup home page
+  app.get('/', function (req, res) {
+    res.render('hello');
+  });
+
+    // Setup the configure tab, with first and second as content tabs
+  app.get('/configure', function (req, res) {
+    res.render('configure');
+  });
+
+  // ------------------
+  // SSO demo page
+  app.get('/ssodemo', function (req, res) {
+    res.render('ssoDemo');
+  });
+
+  // Pop-up dialog to ask for additional permissions, redirects to AAD page
+  app.get('/auth/auth-start', function (req, res) {
+    var clientId = config.get("tab.appId");
+    res.render('auth-start', { clientId: clientId });
+  });
+
+  // End of the pop-up dialog auth flow, returns the results back to parent window
+  app.get('/auth/auth-end', function (req, res) {
+    var clientId = config.get("tab.appId");
+    res.render('auth-end', { clientId: clientId });
+  });
+
+  // On-behalf-of token exchange
+  app.post('/getProfile', function (req, res) {
```

Comment:
Why getProfile why not acquireTokenOnBehalfOf ?

Reply:
Renamed it to getProfileOnBehalfOf